dist/formats/email_alert_signup/frontend/schema.json

```diff
@@ -55,10 +55,8 @@
     },
     "details": {
       "type": "object",
-      "additionalProperties": false,
```

Comment:
I believe that removing additionalProperties means that any fields at all are allowed in the details hash (the schema is checked for those that are defined, but others are allowed) - I don't think this is what we want.